library/src/main/java/com/telefonica/mistica/list/ListRowView.kt

```diff
@@ -238,14 +238,14 @@ open class ListRowView @JvmOverloads constructor(
                 R.styleable.ListRowView_listRowHeadlineLayout,
                 TypedValue.TYPE_NULL
             )
-            val headlineVisible: Boolean = styledAttrs.getBoolean(
-                R.styleable.ListRowView_listRowHeadlineVisible,
-                currentHeadlineLayoutRes != HEADLINE_NONE
-            )
             setHeadlineLayout(
                 layoutRes = headlineResId.takeIf { it != TypedValue.TYPE_NULL } ?: HEADLINE_NONE,
                 contentDescription = styledAttrs.getString(R.styleable.ListRowView_listRowHeadlineContentDescription)
             )
+            val headlineVisible: Boolean = styledAttrs.getBoolean(
+                R.styleable.ListRowView_listRowHeadlineVisible,
+                currentHeadlineLayoutRes != HEADLINE_NONE
+            )
```

Comment:
We can make it more explicit to avoid this error in future refactors, something like that:
```suggestion
setHeadlineLayout(
                layoutRes = headlineResId.takeIf { it != TypedValue.TYPE_NULL } ?: HEADLINE_NONE,
                contentDescription = styledAttrs.getString(R.styleable.ListRowView_listRowHeadlineContentDescription)
            ).also {
                // Visibility decision depends on the title layout initialization
                val headlineVisible: Boolean = styledAttrs.getBoolean(
                    R.styleable.ListRowView_listRowHeadlineVisible,
                    currentHeadlineLayoutRes != HEADLINE_NONE
                )
                setHeadlineVisible(headlineVisible)
            }
```

WDYT?

Reply:
Done [d389f24](https://github.com/Telefonica/mistica-android/pull/385/commits/d389f241bbdd1ced2438480bebe4689a6a378ec2)

Reply:
Sorry, there was an error in the comment, can you please update it? 
// Visibility decision depends on the [~title~ -> **headline**] layout initialization

Reply:
Yup, I saw it, np